kube_helpers.go

```diff
@@ -480,12 +480,14 @@ func getHelmsmanReleases() map[string]map[*release]bool {
 				continue
```

Comment:
How about using the approach I took on https://github.com/Praqma/helmsman/blob/738d7d6905616d5e5e71a93e422e31c33ba955fa/kube_helpers.go#L398 ? It simplifies parsing and while I completely understand the need of fixing this asap before helmsman 3 goes on, this would help us merge it in the future :-) If you follow the `-o name`, then rest of changes I did while migrating to helm 3 simplifies the code of this whole function

Reply:
Sure, I'll update the pr. Thanks

Reply:
I've updated the PR to follow your approach.